### PR TITLE
Added "Failed To Bind To Port" Section to the FAQ

### PR DESCRIPTION
**Pull request by stefmmm**

### CODE REVIEW

**Comment by stefmmm** [comment]
Needs Reworking

**Comment by stefmmm** [comment]
Dunno whether this will include my new changes but here goes.

File: docs/faq.md [hunk 1]
```diff
@@ -5,6 +5,7 @@
 *   [Error occurred during initialization of VM Could not reserve enough space for 2097152KB object heap.](#error-occurred-during-initialization-of-vm)
 *   [My server keeps saying starting.](#server-keeps-saying-starting)
 *   [Unable to connect to the server.](#unable-to-connect-to-the-server)
+*   [Failed to bind to port](#failed-to-bind-to-port)
```
Comment (vaio2005) [inline docs/faq.md:8]: All this part needs to be removed to comply with #17

**Review by vaio2005 — CHANGES_REQUESTED** [review]
Little change, not something big

File: docs/faq.md (hunk 1 above)
Comment (vaio2005) [reply docs/faq.md:8]: Might want to wait for this. Other change could be required based on #17

**Comment by stefmmm** [comment]
Okay sounds good just let me know once you know more.

Sent from ProtonMail mobile

-------- Original Message --------

**Comment by Fiahblade** [comment]
@stefmmm Thanks!

**Pull request merged**